admin-dev/themes/new-theme/js/app/pages/stock/mixins/product-desc.js

```diff
@@ -36,11 +36,13 @@ export default {
       return null;
     },
     combinationName() {
-      const arr = this.product.combination_name.split(',');
+      const combinations = this.product.combination_name.split(',');
+      const attributes = this.product.attribute_name.split(',');
```

Comment:
the attributes are in the format `toto, tata, titi`
So if you split by ",", the white space remains, right ?

Reply:
Fixed ;)